cli/src/diff_util.rs

```diff
@@ -73,15 +75,18 @@ pub struct DiffFormatArgs {
     /// Generate diff by external command
     #[arg(long)]
     pub tool: Option<String>,
+    /// Number of lines of context to show.
+    #[arg(long)]
+    context: Option<usize>,
```

Comment:
Since we seem to use the same default for all formats, and we don't seem to care if it's unset either, can we simply set the default to "3" and make this not an `Option`?

Reply:
Perhaps, that depends on whether we'll add config knob or not. I personally don't need it, but git appears to have one.

Reply:
Let's leave as is. It's an easy refactor if it's decided it's not needed.